lib/kernel/test/socket_SUITE.erl

```diff
@@ -22045,7 +22045,7 @@ api_opt_ipv6_recvpktinfo_udp(InitState) ->
 %% There seem to be some weirdness with the definition of this
 %% option, so its defined in an include file we don't include
 %% (directly or indirectly). And since some of the defines
-%% are occure in a file we *do* include (via netinet/in.h), we
+%% are occurred in a file we *do* include (via netinet/in.h), we
```

Comment:
```suggestion
%% occurred in a file we *do* include (via netinet/in.h), we
```

Reply:
Please remove the "are".

Reply:
> Please remove the "are".

@bjorng Done.

Reply:
Shound't it be "occur" instead? Present tense...

Reply:
@zuiderkwast You are right. Present tense makes more sense.

@kianmeng Please change.

Reply:
@bjorng @zuiderkwast :ok_hand: